Sources/QuickTests/FunctionalTests/ItTests.swift

```diff
@@ -9,16 +9,16 @@ class FunctionalTests_ItSpec: QuickSpec {
     override func spec() {
         var exampleMetadata: ExampleMetadata?
         beforeEach { metadata in exampleMetadata = metadata }
-        
```

Comment:
My bad on all the whitespace in this file... :sweat_smile: 

I fixed it though! :smile: